redbot/core/config.py

```diff
@@ -151,6 +151,64 @@ def __call__(self, default=...) -> _ValueCtxManager[Any]:
         """
         await self.driver.set(*self.identifiers, value=value)
 
+    async def inc(self, value: int) -> int:
+        """Increment and return the value of the data element pointed to by `identifiers`.
+
+        Example
+        -------
+        ::
+
+            # Increments and returns a global value "foo" by 3.
+            new = await conf.foo.inc(3)
+
+            # You can also decrement a value using by negative numbers
+            new = await conf.foo.inc(-3)
+
+            # Floats are also supported
+            new = await conf.foo.inc(3.834)
+
+        Parameters
+        ----------
+        value : int
+            Integer that will be used to increment a value.
+
+        Returns
+        -------
+        `int` or `float`
+            The stored value plus the value given as a integer or float.
+
+        """
+        if isinstance(self.default, bool) or not isinstance(self.default, (int, float)):
+            raise TypeError("Config inc method only supports integer or float values.")
```

Comment:
Might also want to say that the method only supports *registered* integer or float values